pkg/transaction/monitor.go

```diff
@@ -21,6 +21,9 @@ import (
 var ErrTransactionCancelled = errors.New("transaction cancelled")
 var ErrMonitorClosed = errors.New("monitor closed")
 
+// MaxBlocksToWait is the maximum number of blocks to wait before considering a missing transaction as cancelled.
+const MaxBlocksToWait = 5
```

Comment:
This variable does not need to be exported. It can be exported for tests only in export_test.go.